README.md

```diff
@@ -17,11 +17,18 @@ OpenSearch upgrade, migration, and comparison tooling facilitates OpenSearch mig
 
 ## Supported cluster versions and platforms
 
-There are numerous combinations of source clusters, target clusters, and platforms. While the tools provided in this repository might work with various combinations, they might not support breaking changes between different source and target versions. Below is a growing support table that will be updated over time.
-
-|Supported Sources|Supported Targets|Source Platform|Target Platform|
-|:---------------:|:---------------:|:-------------:|:-------------:|
-|ElasticSearch 7-7.10.2|OpenSearch 1.x and 2.x|AWS Self-Managed|AWS OpenSearch and OpenSearch Serverless|
+There are numerous combinations of source clusters, target clusters, and platforms. While the tools provided in this repository might work with various combinations, they might not support breaking changes between different source and target versions. Below is a list of supported source and target versions and platforms.
+
+### Supported Source and Target Versions
```

Comment:
I like this view better, but I think that once we have integ tests in place, it will be really nice to list a table of specific combinations that we're testing.  So, this works for now and later but I'd like to see the table (or a link to one) come back in in the coming weeks as we add tests.

Reply:
I would love to see that. I've been discussing this with @lewijacn